doc/en/user/source/extensions/authkey/index.rst

```diff
@@ -237,6 +237,31 @@ After configuring the filter it is necessary to put this filter on the authentic
    saves the current configuration and triggers a synchronize. If users are added/removed from 
    the backing user/group service, the synchronize logic should be triggered.
 
+Challenge Anonymous Session
+---------------------------
+
+The AuthKey filter now includes a new option, "Challenge Anonymous Sessions," designed to enforce stricter authorization and stateless behavior.
```

Comment:
This is not an announcement... in a few months there will be nothing new about this functionality any more. Best to just state what it does and use present tense

```suggestion
"Challenge Anonymous Sessions" is designed to enforce stricter authorization and stateless behavior.
```